CONTRIBUTING.md

```diff
@@ -1,3 +1,14 @@
+# 👨‍💻 Prerequisite Skills to Contribute in public/Profile.
+- [Git](https://git-scm.com/)
+- [JSON](https://www.json.org/json-en.html)
+
+# 👨‍💻 Prerequisite Skills to Contribute in Documents.
+- [Markdown](https://www.markdownguide.org/basic-syntax/)
+
+# 👨‍💻 Prerequisite Skills to Contribute in Components/CSS.
+- [React](https://reactjs.org/)
+- [React Prime](https://www.primefaces.org/primereact/)
```

Comment:
For CSS, shouldn't we mention some documentation for CSS/HTML?

Reply:
@Panquesito7 the css should preferably come from prime react with as little vanilla css as possible